bin/deploy-vhost

```diff
@@ -135,16 +135,30 @@ sub get_pgpw_string {
 
 sub one_db_conf_php {
     my ($params, $database, $internal) = @_;
-    my $port_line = '';
+    my $port_line       = '';
```

Comment:
This line didn't change ;) `not recommended` is our coding standard on this, but only if I make another comment about something else actual to change :)

Reply:
It's removed by the 2nd commit anyway.